buildbot/master/files/config/factories.py

```diff
@@ -192,7 +208,10 @@ def make_step(self, command):
             # Provide environment variables for s3cmd
             elif arg == './etc/ci/upload_nightly.sh':
                 step_kwargs['logEnviron'] = False
-                step_env = copy.deepcopy(envs.upload_nightly)
+                if is_windows():
+                    step_env = copy.deepcopy(envs.upload_nightly_windows)
+                else:
+                    step_env = copy.deepcopy(envs.upload_nightly)
```

Comment:
We should just do `step_env += envs.upload_nightly` and get rid of `envs.upload_nightly_windows` - would have mentioned this on the earlier PR if it wasn't right before release :)